sdk/core/Azure.Core/src/Shared/NextLinkOperationImplementation.cs

```diff
@@ -38,7 +44,7 @@ internal class NextLinkOperationImplementation : IOperation
             bool skipApiVersionOverride = false,
             string? apiVersionOverrideValue = null)
         {
-            string? apiVersionStr = null;
+            string? apiVersionStr;
```

Comment:
Curious: why not initialize the local variable?

Reply:
The variable is assigned with the coming lines right away, no point to initialize it. Actually, visual studio recommends to remove it.

Reply:
I did not touch the logic in current PR, I can revert the change if you think it is needed.

Reply:
reverted